packages/common/src/projects/edit.ts

```diff
@@ -153,3 +182,31 @@ export async function getHubProjectEditorConfig(
 
   return Promise.resolve({ schema, uiSchema });
 }
+
+/**
+ * Takes the resources prop from a projects model and returns an array of
+ * objects with a filename and resource to be created.
+ *
+ * @param {*} resources
+ * @return {*}
+ */
+/* istanbul ignore next */
+function configureProjectResources(resources: any) {
+  // Set up obj tracking resource prop && filename (KEEP IN SYNC WITH getPropertyMap()
+  // for additional resources!)
+  const resourceNameAndProp: {
+    [key: string]: string;
+  } = {
+    location: "location.json",
+  };
```

Comment:
Per @dbouwman we want to be very explicit about prop/filename matching. We should consider a centralized location to keep this obj, the one in fetch in sync along with the prop names in getBasePropertyMap.